src/CMake/version.cmake

```diff
@@ -4,7 +4,7 @@
 # Get the branch
```

Comment:
Copyright date (line 2) needs to be updated.  Being that we are AMD, you might wish to add the following line:
`Copyright (C) 2022 Advanced Micro Devices, Inc.`